net-analyzer/nmbscan/nmbscan-1.2.5-r1.ebuild

```diff
@@ -0,0 +1,27 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="netbios scanner"
```

Comment:
Maybe we should use the `NetBIOS` here, it seems to be the correct capitalisation.

Reply:
I've updated the ebuild(s) and improved the description. While i was on it i've also fixed the SRC_URI and HOMEPAGE. (for the existing ebuild too)